convert_results.py

```diff
@@ -0,0 +1,182 @@
+"""This script converts JSON result files to Markdown summaries.
+
+The JSON format is meant to contain all details from tests. For human readable
+outputs, use this script to extract the most interest information from your JSON
+reports and write them into an eye-friendly markdown file.
+
+  Typical usage example:
+
+  python convert_results.py --source=results/ --target=result_summaries/
+"""
+
```

Comment:
You might consider refactory the results/ directory so we have
results/json/
results/summary.md
results/script/thisscript.py
results/md/
results/assets/ (if we ever do visz)

Reply:
I wanted to keep it general since we have `results/` and `results_fuzzing/` with JSON of the same format. Changing the directory structure is fine, we just have to make sure it works for multiple results directories.

Reply:
I decided to put the summaries into the results folder by default as suggested. To keep it general, you can set any input and output directory, including nesting.
We will have 2 results folders, one for fuzzing and one for regular. Therefore, the template and this script stay outside of `results/`.
`assets/` and `summary/` can be added in a later PR.